### ISSUE TITLE
Packages don't emit events to indicate they installed successfully

### ISSUE THREAD

**Issue by negz** [crossplane/crossplane#1747]
### What happened?

I noticed while installing a (v2) Provider package that while its Provider installed successfully, the Provider never gave any indication that it worked:

```bash
$ kubectl -n crossplane-system describe providerrevision.pkg|tail -n4
Events:
  Type    Reason          Age                From                                         Message
  ----    ------          ----               ----                                         -------
  Normal  InstallPackage  1s (x5 over 2m9s)  packages/providerrevision.pkg.crossplane.io  Successfully installed package revision
```

```bash
$ kubectl -n crossplane-system describe provider.pkg|tail -n16
Status:
  Conditions:
    Last Transition Time:  2020-09-19T04:30:26Z
    Reason:                ActivePackageRevision
    Status:                True
    Type:                  Installed
    Last Transition Time:  2020-09-19T04:30:24Z
    Reason:                HealthyPackageRevision
    Status:                True
    Type:                  Healthy
  Current Revision:        provider-aws-ad48c13c2b38
Events:
  Type     Reason         Age    From                                 Message
  ----     ------         ----   ----                                 -------
  Normal   UnpackPackage  4m52s  packages/provider.pkg.crossplane.io  waiting for unpack to complete
  Warning  UnpackPackage  4m52s  packages/provider.pkg.crossplane.io  pods "hasheddan-xp3-provider-aws-v0.10.0" already exists
```

Ideally the provider would let me know via an event when a revision bcomes active.

### How can we reproduce it?

Apply the following against 15d496349f366685dbe1d53d8726562874722a5d:

```yaml
apiVersion: pkg.crossplane.io/v1alpha1
kind: Provider
metadata:
  name: provider-aws
spec:
  package: hasheddan/xp3-provider-aws:v0.10.0
```

### What environment did it happen in?
Crossplane version:

**Comment by negz** [comment]
CC @hasheddan - Nice to have for v0.13, but not critical.

**Comment by hasheddan** [comment]
@negz definitely will get this fixed for v0.13, thanks!
